src/modules/map/constants.js

```diff
@@ -1,9 +1,9 @@
 import {normalizeActionName} from '../common/helpers.js';
 
-export const MAP_URL = 'https://geoserver.hel.fi/mapproxy/wmts/osm-lite/etrs_tm35fin/{z}/{x}/{y}.png';
-export const MAP_RETINA_URL = 'https://geoserver.hel.fi/mapproxy/wmts/osm-lite-hq/etrs_tm35fin_hq/{z}/{x}/{y}.png';
+export const MAP_URL = 'http://tiles.hel.ninja/styles/hel-osm-light/{z}/{x}/{y}.png';
```

Comment:
Preferably use `https`.

Reply:
Oops. Thats what you get for copypasting from helpful demo interface.